src/io/reader.rs

```diff
@@ -4,6 +4,7 @@ use std::path::Path;
 
 use crate::dynimage::DynamicImage;
 use crate::image::ImageFormat;
+use crate::open; // docs only
```

Comment:
Is this fix supposed to address this missing link?

> https://docs.rs/image/0.23.2/image/io/struct.Reader.html#usage-1

If so, that way only works on nightly as automatic link resolution is not yet stable. It seems that the link target was simply forgotten. In any case, it's supposed to refer to `Reader::open` and not `crate::open`.

Reply:
Thanks, I've pushed another commit to fix this (i.e. pointing at `Reader::open` instead).

However I hadn't realized that the intra-rustdoc links are not yet stable. Thanks. Are you interested in progressing this PR anyway? Fortunately it looks like [docs.rs is built with nightly](https://docs.rs/about) so all the various links I've splurged in will hopefully work in that context.

Alternatively, if you don't want to accept this PR currently, would you accept a smaller one to add a cross-link from `src/lib.rs` to the github page which displays `README.md`?